deploy/addons/gcp-auth/gcp-auth-webhook.yaml.tmpl.tmpl

```diff
@@ -41,7 +41,7 @@ rules:
       - update
 
 ---
-apiVersion: rbac.authorization.k8s.io/v1beta1
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
which version of kubernetes has `apiVersion: rbac.authorization.k8s.io/v1` ?
would this mean older kubernetes versions wont be able to use this addon ?

Reply:
v1beta1 has been deprecated since 1.17 and has started causing failures in newer versions of k8s, this is really the only answer here.